src/freenas/usr/local/lib/middlewared_truenas/plugins/failover.py

```diff
@@ -467,7 +487,12 @@ def _disabled_reasons(self, app):
                 reasons.append('NO_LICENSE')
 
             local = self.middleware.call_sync('failover.vip.get_states')
-            remote = self.middleware.call_sync('failover.call_remote', 'failover.vip.get_states')
+            try:
+                remote = self.middleware.call_sync('failover.call_remote', 'failover.vip.get_states')
+            except Exception:
```

Comment:
We should find out which exception exactly it is, we can identify if the method was really not found or another error.